src/business-components/CountryFlag/index.js

```diff
@@ -1,32 +1,43 @@
 import React from 'react';
 import { Tooltip, withStyles } from '@material-ui/core';
 import ReactCountryFlag from 'react-country-flag';
+import ISO3166_1 from "iso-3166-1";
```

Comment:
Check whether we can use something like "isValidISOCode" in existing  "react-country-flag" package rather than install a new package

import { isValidISOCode } from "react-country-flag";

if (isValidISOCode(newCountryCode)) {
      //something
}